packages/meditrak-app/app/database/types/Entity.js

```diff
@@ -42,14 +44,17 @@ Entity.schema = {
     parent: { type: 'Entity', optional: true },
     code: { type: 'string', optional: true },
     type: { type: 'string', default: 'Entity not properly synchronised' },
+    attributesType: { type: 'string', optional: true },
```

Comment:
I'm aware that it's better to sync the whole `attributes` column. But as I understand it requires creating a new schema for it (in order to perform filtering) as well as link it to Entity in order to update whenever entity is sync. 

Maybe, if we want to go with syncing the whole attributes column, what I can do is in Attributes schema, have an entity_id property that points to the Entity it is attached to so that when syncing we can find the corresponding attributes and update?